docs/source/migrate_to_notebook7.md

```diff
@@ -1,27 +1,62 @@
 # Migrating to Notebook 7
 
-```{warning}
-The Jupyter Notebook application is undergoing a major refactoring to
-improve the user experience and to make it easier to maintain and extend.
+*Updated 2023-04-24*
 
-This set of migration guides are intended to help you migrate your Classic Notebook
-setup and extensions to the new Jupyter Notebook 7, which is built on top of JupyterLab 4.
+```{warning}
+The Jupyter Notebook application's newest version 7 upgrade might break your
+extensions or other customizations, please read more below to find out if you
+need to take any actions to ensure a smooth, uninterrupted experience
 ```
 
-## Sunrising the Jupyter Notebook 7
+A major upgrade to the Jupyter Notebook app is coming with Notebook 7! This
+upgrade will bring a heap of new features, but will also break backwards
+compatibility with many classic Notebook features and customizations.
+
+This set of guides is here to help you migrate your Classic Notebook setup and
+extensions to the new Notebook 7.
+
+## What you need to do
+
+For users who don't use extensions or other customizations, you will seamlessly
+receive the new Notebook 7 when you `pip install notebook` once version 7 is
+released out of beta, along with all its new features, like realtime
+collaboration, debugger, and theming.
+
+For users who need to use extensions or other customizations, you have a couple
+of options:
+
+- Look for Notebook 7 compatible versions of the extensions you already use,
+  and [find replacements for those that are not available]
+
+- If you NEED to maintain compatibility with the Classic Notebook for extensions
+  or other customizations that are critical to your workflows, you can switch to
+  [nbclassic], the new home for users who need to maintain compatibility with the
```

Comment:
The "new" and "maintain" words in this sentence give the somewhat misleading idea that nbclassic will be maintained in the long run?

NBClassic is supposed to help with the transition to Notebook 7, but shouldn't be seen as an "alternative" to Notebook 7.

Reply:
Yeah it's a subtle thing to communicate. We want people who need it to know it's there and to use it, without giving the impression it's a permanent thing. I'll add a bit/re-work to add some clarity. Thanks!

Reply:
I posted a comment in the team compass issue to see if the EC can provide guidance for the documentation as well, since they will be looking at the blog post: https://github.com/jupyter/notebook-team-compass/issues/24#issuecomment-1521772403

Reply:
I have updated this portion with a recent commit, and we can modify as needed after the EC takes a look.